core/src/fetched-record-buffer.ts

```diff
@@ -7,14 +7,13 @@ export class FetchedRecordBuffer {
     this.store = {}
   }
 
-  static fromStore(store: { [subject: string]: { [position: string]: FetchedRecord[] } }) {
-    const buffer = new FetchedRecordBuffer()
+  static fromStore(store: { [subject: string]: { [position: string]: FetchedRecord[] } }) {    const buffer = new FetchedRecordBuffer()
```

Comment:
It looks like this got combined into a single line

Reply:
Sorry, I didn't notice. This is updated.